cfme/tests/containers/test_ssl_providers.py

```diff
@@ -49,9 +48,38 @@
 )
 
 
+@pytest.fixture(scope="module")
+def sync_ssl_certificate(provider, appliance):
+    """
+    fixture which sync SSL certificate between CFME and OCP
+    :param provider:  OCP provider object
+    :param appliance: CFME appliance object
+    """
+
+    provider_ip = provider.data["ipaddress"]
+    provider_cred_name = provider.data["credentials"]
```

Comment:
Thoughts on this instead since `provider_cred_name` is not used anywhere else:
`provider_cred_password = provider.get_credentials_from_config(provider_cred_name).secret`

and you can set `password` below to `provider_cred_password'

Reply:
I don't understand this comment, can you explain again, please?

Reply:
Sure, happy to explain. So I see that you are setting `provider_cred_name = provider.data["credentials"]`, but it is only used once (on line 65). So I was thinking, to shorten line 65, you could set `provider_cred_password = provider.get_credentials_from_config(provider_cred_name).secret` instead of `provider_cred_name = provider.data["credentials"]`. Then on line 65, set `"password": provider_cred_password,`. This is only a suggestion.